react-components/package.json

```diff
@@ -4,18 +4,18 @@
   "homepage": "/de/react_components",
   "private": true,
   "devDependencies": {
-    "@storybook/addon-actions": "^3.4.8",
-    "@storybook/addon-console": "^1.0.0",
-    "@storybook/addon-links": "^3.4.8",
-    "@storybook/addons": "^3.4.8",
-    "@storybook/react": "^3.4.8",
+    "@storybook/addon-actions": "^4.1.0",
+    "@storybook/addon-console": "^1.1.0",
+    "@storybook/addon-links": "^4.1.0",
+    "@storybook/addons": "^4.1.0",
+    "@storybook/react": "^4.1.0",
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
     "jest-junit-reporter": "^1.1.0",
     "prop-types": "^15.6.2",
     "react-scripts": "^1.0.14"
   },
   "dependencies": {
-    "@material-ui/core": "^3.0.1",
+    "@material-ui/core": "^3.5.0",
```

Comment:
When I attempted `npm install --save @material-ui/core@3.5.0`, I got the following warning:
```
npm WARN deprecated @material-ui/core@3.5.0: This version is broken, use @material-ui/core@v3.5.1 instead
```
We should bump this up to at least v3.5.1 before merging this PR (they are currently at v3.9.0).

I also saw the following warning:
```
npm WARN @material-ui/icons@2.0.0 requires a peer of @material-ui/core@^1.0.0 but none is installed. You must install peer dependencies yourself.
```
So I think we can also bump `@material-ui/icons` to v3.0.2.